main.py

```diff
@@ -7,6 +7,7 @@
 from PIL import Image
 from aiohttp import web
 from collections import UserDict
+import requests
```

Comment:
please god no, there is a reason requests isnt alreay imported

look at some other commands (/random, /tiktok etc) to see how to use aiohttp

Reply:
alright